### ISSUE TITLE
Tree list items, refresh and more button is not accessible via keyboard tab key

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#137717]
GitHubTags:#A11yMAS;#A11yTCS;#A11ySev1;#Keyboard;#Desktopweb;#WCAG2.1.1;#A11yAuto;#Win10;#AILimited;#TabStop;#Benchmark;#FTP;#A11y_VSCode;#CAITestPass_May2020;#Visual Studio Code;

## Environment Details:
#URL: https://ms.portal.azure.com/#home
Chromium edge: Version 95.0.1020.44 (Official build) (64-bit)

## Repro Steps:

1. Open URL https://ms.portal.azure.com/
2. Navigate to the cloud shell button on the top menu bar and select it.
3. Navigate to "Open Editor" button of "CloudShell-PowerShell Editor" window and select it
4. press tab to move to the editor and then, press tab and ctrl+M
5. use shift +tab focus goes on the entire tree list but not individual items

## Actual:
Tree views, refresh and more button is not accessible via keyboard tab key. on shift+tab to the tree list the arrow keys do not move to the items.

## Expected Result:
Tree views, refresh and more button should accessible via keyboard tab and arrow keys

## User Impact:
keyboard users will not be able to access the controls

## Attachment:
![140724690-c3407f7d-b714-468e-be1d-1e9f535907a0](https://user-images.githubusercontent.com/24580570/143035591-e2037a87-9bc0-40ea-b840-adcc7b781bbb.gif)

**Comment by chrmarti** [comment]
/duplicate https://github.com/microsoft/monaco-editor/issues/2753
